documentation/sections/breaking_changes_argument_parsing_and_shebang_lines.md

```diff
@@ -11,6 +11,25 @@ Usage: coffee [options] [--] path/to/script.coffee [args]
 
 If called without options, `coffee` will run your script.
 
+  -b, --bare         compile without a top-level function wrapper
+  -c, --compile      compile to JavaScript and save as .js files
+  -e, --eval         pass a string from the command line as input
+  -h, --help         display this help message
+  -i, --interactive  run an interactive CoffeeScript REPL
+  -j, --join         concatenate the source CoffeeScript before compiling
+  -m, --map          generate source map and save as .js.map files
+  -M, --inline-map   generate source map and include it directly in output
+  -n, --nodes        print out the parse tree that the parser produces
+      --nodejs       pass options directly to the "node" binary
+      --no-header    suppress the "Generated by" header
+  -o, --output       set the output directory for compiled JavaScript
+  -p, --print        print out the compiled JavaScript
+  -r, --require      require the given module before eval or REPL
+  -s, --stdio        listen for and compile scripts over stdio
+  -l, --literate     treat stdio as literate style coffeescript
+  -t, --tokens       print out the tokens that the lexer/rewriter produce
+  -v, --version      display the version number
+  -w, --watch        watch scripts for changes and rerun commands
```

Comment:
That makes sense. I have noted only the changed usage line now and left an `...` to denote the rest of the help text.